File: examples/verify.py
from libpasta import *

class User:
    password_hash = ""

    def auth_user():
        password = read_password("Enter password:")
        if verify_password(self.password_hash, password):
            print("The password is correct!")
            # ~> Handle correct password
        else:
            print("Incorrect password.")
            # ~> Handle incorrect password

File: examples/verify.java
import io.github.libpasta.pasta;

public class verify {
    private String password_hash;

    public void authUser() {
        String password = pasta.read_password("Enter password:");
        if(pasta.verify_password(self.password_hash, password)) {
            System.out.println("The password is correct!");
            // ~> Handle correct password
        } else {
            System.out.println("Incorrect password.");
            // ~> Handle incorrect password
        }
    }
}
